packages/gatsby/src/joi-schemas/joi.js

```diff
@@ -24,12 +24,14 @@ export const pageSchema = Joi.object()
 export const nodeSchema = Joi.object()
   .keys({
     id: Joi.string().required(),
-    contentDigest: Joi.string().required(),
     children: Joi.array(Joi.string()).required(),
     parent: Joi.string().required(),
-    mediaType: Joi.string().required(),
-    content: Joi.string(),
-    type: Joi.string().required(),
-    pluginName: Joi.string().required(),
+    internal: Joi.object().keys({
+      contentDigest: Joi.string().required(),
+      mediaType: Joi.string().required(),
+      type: Joi.string().required(),
+      pluginName: Joi.string().required(),
+      content: Joi.string(),
+    }),
```

Comment:
If it's not required according to this validation, shouldn't `contentDigest` then not be required too? Or are those not added in the same way?